src/SFA.DAS.AANHub.Application/Apprentices/Queries/GetApprenticeMemberQueryValidator.cs

```diff
@@ -1,14 +1,22 @@
 ﻿using FluentValidation;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
 
 namespace SFA.DAS.AANHub.Application.Apprentices.Queries
 {
     public class GetApprenticeMemberQueryValidator : AbstractValidator<GetApprenticeMemberQuery>
     {
-        public GetApprenticeMemberQueryValidator()
+        public const string ApprenticeIdNotFoundMessage = "ApprenticeId was not found";
+        public GetApprenticeMemberQueryValidator(IApprenticesReadRepository apprenticesReadRepository)
         {
             RuleFor(a => a.ApprenticeId)
                 .NotNull()
-                .NotEmpty();
+                .NotEmpty()
+                .MustAsync(async (apprenticeid, cancellation) =>
```

Comment:
I think we agree that there should not be a async check on Get as this will return BadRequest where as we want to return NotFound.